tools/python/scell_seal_string_echo_pw.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020 Cossack Labs Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+from base64 import b64encode, b64decode
+from pythemis import scell
+
+_, COMMAND, KEY, MESSAGE, CONTEXT = range(5)
+
+if len(sys.argv) not in (4, 5):
+    sys.stderr.write('usage: {enc|dec} <passphrase> <message> [context]\n')
```

Comment:
p.s. one more way to write to stderr is `print('text', file=sys.stderr)`. It write "\n" at end by default.

Reply:
> p.s. one more way to write to stderr is `print('text', file=sys.stderr)`

It would be better, but it does not work in Python 2 :(